.yarnrc

```diff
@@ -0,0 +1 @@
+--add.exact true
```

Comment:
^^

Reply:
It makes Yarn automatically pin dependencies when you do a regular `yarn add <package>`.

Reply:
yas.. I figured.